src/mmw/js/src/modeling/models.js

```diff
@@ -34,10 +34,15 @@ var ProjectModel = Backbone.Model.extend({
         area_of_interest: null,    // GeoJSON
         model_package: '',         // Package name
         taskModel: null,           // TaskModel
-        scenarios: null            // ScenariosCollection
+        scenarios: null,            // ScenariosCollection
+        uid: null
     },
 
     initialize: function() {
+        var scenarios = this.get('scenarios');
+        if (scenarios === null || typeof scenarios === 'string') {
```

Comment:
Seems like we should have complete control during Project initialization.  What are the cases where this is null or a string?  If we're doing a custom parse on load, I'm not sure why this is necessary.